cfme/tests/control/test_actions.py

```diff
@@ -624,13 +625,7 @@ def wait_analysis_tried():
     def is_vm_analysis_finished():
         """ Check if analysis is finished - if not, reload page
         """
-        tab_name = pick({
-            LOWEST: "All VM Analysis Tasks",
-            '5.6': "All VM and Container Analysis Tasks",
-        })
-        if not pytest.sel.is_displayed(tasks.tasks_table) or \
-           not tabs.is_tab_selected(tab_name):
-            pytest.sel.force_navigate('tasks_all_vm')
+        navigate_to(Tasks, 'AllVMContainerAnalysis')
```

Comment:
this makes me think the next step should be a view thats returned on task navigation instead of the global table